src/TestFramework/TestFramework.Extensions/MSTest.TestFramework.nuspec

```diff
@@ -84,6 +84,7 @@
     <file src="net6.0-windows10.0.18362.0\Microsoft.VisualStudio.TestPlatform.TestFramework.Extensions.dll" target="buildTransitive\net6.0\winui\" />
     <file src="net6.0-windows10.0.18362.0\Microsoft.VisualStudio.TestPlatform.TestFramework.Extensions.xml" target="buildTransitive\net6.0\winui\" />
     <file src="$RepoRoot$src\TestFramework\TestFramework.Extensions\build\net6.0\MSTest.TestFramework.targets" target="build\net6.0\" />
+    <file src="$RepoRoot$src\TestFramework\TestFramework.Extensions\build\net6.0\MSTest.TestFramework.targets" target="buildTransitive\net6.0\" />
```

Comment:
buildTransitive should come from `net6.0\buildTransitive\winui+uwp\MSTest.TestFramework.targets` which is already the case for net6.0, but isn't done for all TFMs, specifically, .NET Standard and .NET Framework.